components/clientComponents/forms/Review/Review.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
```

Comment:
TODO: Come back to this. The build suddenly gave an error about there being no parent with a "use client". The useRef was highlighted as the cause. Why now suddenly? Also the parent does have a "use client".